mindsdb/integrations/handlers/mariadb_handler/Manual_QA.md

```diff
@@ -34,4 +34,60 @@ Drop a remark based on your observation.
 - [ ] Works Great 💚 (This means that all the steps were executed successfuly and the expected outputs were returned.)
 - [ ] There's a Bug 🪲 [Issue Title](URL To the Issue you created) ( This means you encountered a Bug. Please open an issue with all the relevant details with the Bug Issue Template)
 
----
\ No newline at end of file
+---
+
+
+## Testing MariaDB Handler with [All Nobel Laureates 1901-Present](https://www.kaggle.com/datasets/prithusharma1/all-nobel-laureates-1901-present?resource=download)
+
+**1. Testing CREATE DATABASE**
+
+```
+CREATE DATABASE mysql_datasource
+WITH ENGINE = 'mariadb'
+PARAMETERS = {
+  "user": "root",
+  "port": 11433,
+  "password": "[password]",
+  "host": "0.tcp.ngrok.io",
+  "database": "mysql"
+};
+```
+
+![CREATE_DATABASE](https://github.com/wunzt/mindsdb/assets/102569472/92a5ce8f-a98d-42dd-8aab-2a35a9859abd)
+
+
+**2. Testing CREATE PREDICTOR**
+
+```
+CREATE MODEL mindsdb.nobel_model2
+FROM mysql_datasource
+(SELECT * FROM mysql_datasource.nobel_latest2)
+PREDICT prize_share;
+```
+
+![CREATE_PREDICTOR](https://github.com/wunzt/mindsdb/assets/102569472/74075bd0-1735-450f-aefd-10f1f92039d6)
+
+
+**3. Testing SELECT FROM PREDICTOR**
+
+```
+SELECT category
+FROM mindsdb.nobel_model2
+WHERE category='peace';
+```
+
+(No Image Due To Error During CREATE PREDICTOR)
```

Comment:
Please find out the error message and report it in the `Results` section at the bottom. You can create a bug issue.